taskt/Core/Automation/Commands/Outlook Commands/MoveCopyOutlookEmailsCommand.cs

```diff
@@ -141,18 +135,19 @@ public override void RunCommand(object sender)
         public override List<Control> Render(frmCommandEditor editor)
         {
             base.Render(editor);
-            RenderedControls.AddRange(CommandControls.CreateDefaultDropdownGroupFor("v_OperationType", this, editor));
+
             RenderedControls.AddRange(CommandControls.CreateDefaultInputGroupFor("v_SourceFolder", this, editor));
             RenderedControls.AddRange(CommandControls.CreateDefaultInputGroupFor("v_Filter", this, editor));
-            RenderedControls.AddRange(CommandControls.CreateDefaultDropdownGroupFor("v_MoveUnreadOnly", this, editor));
             RenderedControls.AddRange(CommandControls.CreateDefaultInputGroupFor("v_DestinationFolder", this, editor));
+            RenderedControls.AddRange(CommandControls.CreateDefaultDropdownGroupFor("v_OperationType", this, editor));
+            RenderedControls.AddRange(CommandControls.CreateDefaultDropdownGroupFor("v_MoveCopyUnreadOnly", this, editor));
 
             return RenderedControls;
         }
 
         public override string GetDisplayValue()
         {
-            return base.GetDisplayValue() + $" [From: {v_SourceFolder}, To: {v_DestinationFolder}]";
+            return base.GetDisplayValue() + $" [From '{v_SourceFolder}' - Filter by '{v_Filter}' - {v_OperationType} Emails to '{v_DestinationFolder}']";
```

Comment:
Please change it to `[{v_OperationType} to '{v_DestinationFolder}' From '{v_SourceFolder}' - Filter by '{v_Filter}']`

Reply:
done